pkg/kds/mux/client.go

```diff
@@ -64,6 +65,7 @@ func (c *client) Start(stop <-chan struct{}) (errs error) {
 	}
 	dialOpts := c.metrics.GRPCClientInterceptors()
 	dialOpts = append(dialOpts, grpc.WithUserAgent(version.Build.UserAgent("kds")), grpc.WithDefaultCallOptions(
+		grpc.UseCompressor(gzip.Name),
```

Comment:
PLEASE DON'T MERGE THIS, I'LL DO IT MYSELF AND WILL MONITOR FOR CPU/MEM USAGE - DON'T WANT AN OUTAGE DURING HOLIDAYS